### DISCUSSION TITLE
Sobre escrever o READ.ME completo

### DISCUSSION THREAD

**Discussion by alanafilgueiras** [dcTeam7/Course#106] [Q&A] [answered]
Eu to com uma dúvida. A minha issue engloba todo o read.me certo? 

Eu tenho que esperar o pessoal acabar os tópicos do read.me(issue). 
**Minha dúvida é:** 
Eu também tenho que criar uma branch e alterar o arquivo read.me a medida que for sendo aprovado as PR? Pq eu entendi que depois de revisado e aprovado o merge vai acontecer e a branch "main" atualizar automaticamente.
Comentei sobre isso com  o @KevinDik e ele mesmo sugeriu de ficar registrado o que foi feito por meio de PRs, mas não faço ideia de como fazer isso.
E o passo a passo que eu entendi pra criar uma PR foi esse aqui:

- 1 criar um branch de preferência com a issue lhe foi designada
- 2 alterar o arquivo read.me dentro da sua branch
- 3 fazer o commit 
- 4 fazer uma PR atrelada a issue marcando os revisores

O @joellacerda também ficou responsável pelo READ.ME no in_progress, tem alguma ideia de como fazer? 
@AbraaoAlves

**Comment by joellacerda** [comment, marked as answer]
Pelo que eu entendi, nós que estamos responsáveis pelo README como um todo só precisamos nos preocupar em ver se as issues estão sendo feitas e organizar pra que fique tudo em ordem.
Até porque o arquivo e os commits mesmo vão ser feitos por outras pessoas.

**Reply by alanafilgueiras** [reply]
@joellacerda é, isso que eu imaginei mesmo... vamos revisando.

**Comment by kevinDsousa** [comment]
Acredito que poderia registrar as issues de vocês com os PR apos todos finalizarem as suas tarefas a modo que possa realizar a organização de todo código antes de passar por algum review. Isso levando em conta que sim as tarefas de vocês precisariam ser aberta os PR's.
